extensions/amp-story/1.0/amp-story-page-attachment.css

```diff
@@ -125,3 +125,87 @@ amp-story[desktop] .i-amphtml-story-page-attachment-remote  {
+}
+
+[submit-success] .i-amphtml-story-page-attachment-form-submission-status {
+  color: #0F8514 !important;
+}
+
+[submit-error] .i-amphtml-story-page-attachment-form-submission-status {
+  color: #DE3333 !important;
+}
+
+@keyframes submission-status-fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+[submitting] .i-amphtml-story-page-attachment-form-submission-status,
+.i-amphtml-story-page-attachment-form-submission-status div {
+  /* The entire [submitting] div will fade in along with its border, after which only the contents
+  of the [submit-success] and [submit-error] divs will fade in. */
+  animation: submission-status-fade-in 0.3s !important;
+}
+
+.i-amphtml-story-page-attachment-form-submission-status-icon {
+  height: var(--i-amphtml-story-page-attachment-form-submission-status-icon-height) !important;
+  margin: 0px 14px !important;
+  width: var(--i-amphtml-story-page-attachment-form-submission-status-icon-height) !important;
+}
+
+[submit-success] .i-amphtml-story-page-attachment-form-submission-status-icon {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="26" height="26" fill="none"><rect x="-14" y="-14" width="328" height="54" rx="12" fill="white"/><circle cx="13" cy="13" r="13" fill="#8BDC8F"/><path d="M5.77777 13.8972L10.7342 18.8889L20.2222 9.33334" stroke="black" stroke-width="2.5"/></svg>') !important;
+}
+
+[submit-error] .i-amphtml-story-page-attachment-form-submission-status-icon {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="26" height="26" viewBox="0 0 26 26" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M12.987 0C5.811 0 0 5.824 0 13C0 20.176 5.811 26 12.987 26C20.176 26 26 20.176 26 13C26 5.824 20.176 0 12.987 0ZM14.3 13C14.3 13.715 13.715 14.3 13 14.3C12.285 14.3 11.7 13.715 11.7 13V7.8C11.7 7.085 12.285 6.5 13 6.5C13.715 6.5 14.3 7.085 14.3 7.8V13ZM14.3 18.2C14.3 18.915 13.715 19.5 13 19.5C12.285 19.5 11.7 18.915 11.7 18.2C11.7 17.485 12.285 16.9 13 16.9C13.715 16.9 14.3 17.485 14.3 18.2Z" fill="#FF5252"/></svg>') !important;
+}
+
+.i-amphtml-story-page-attachment-form-submission-status .i-amphtml-story-spinner {
+  height: var(--i-amphtml-story-page-attachment-form-submission-status-icon-height) !important;
+  margin: 0 auto !important;
+  position: relative !important;
+  width: var(--i-amphtml-story-page-attachment-form-submission-status-icon-height) !important;
+
+  /* Overriding default spinner values. */
+  bottom: 0 !important;
+  left: 0 !important;
+  right: 0 !important;
+}
+
+.i-amphtml-story-page-attachment-form-submission-status .i-amphtml-story-spinner-layer {
+  color: black !important;
+
+  /* Overriding default spinner values. */
+  filter: none !important;
+}
+
+[theme="dark"] .i-amphtml-story-page-attachment-form-submission-status .i-amphtml-story-spinner-layer {
+  color: white !important;
```

Comment:
Is there any CSS variable you can use to remove this? Like the font-color maybe?

Reply:
Yup, font-color works! Didn't realize the variables could be reused between files :)